examples/fluids/advection.h

```diff
@@ -59,6 +58,16 @@ struct DCContext_ {
 };
 #endif
 
+#ifndef surface_context_struct
+#define surface_context_struct
+typedef struct SurfaceContext_ *SurfaceContext;
+struct SurfaceContext_ {
+  CeedScalar E_wind;
+  CeedScalar strong_form;
+  PetscBool implicit;
```

Comment:
```suggestion
  bool implicit;
```
It's cleaner not to use Petsc's typedefs in QFunction code, which should be separate and independent from the user-code infrastructure

Reply:
Done. Thanks.